Gemfile

```diff
@@ -15,10 +15,14 @@ gem 'sass-rails',                 '6.0.0'
 gem 'webpacker',                  '4.2.2'
 gem 'turbolinks',                 '5.2.1'
 gem 'jbuilder',                   '2.10.0'
+# pg was only in production
+gem 'pg',                         '1.2.3'
 gem 'bootsnap',                   '1.4.6', require: false
+# I will run autocorrect as last step.
```

Comment:
Usually, if you leave comments for yourself to be resolved later, it's a good idea to prefix them with `TODO`.

Something like:
`TODO: run autocorrect as a last step`

But seeing that you actually forgot to run autocorrect anyway, i'd suggest to not use comments for that purpose and to create separate issues in a repo.